liquidctl/cli.py

```diff
@@ -264,13 +269,28 @@ def main():
 
     if args['--debug']:
         args['--verbose'] = True
-        logging.basicConfig(level=logging.DEBUG, format='[%(levelname)s] %(name)s: %(message)s')
-        _LOGGER.debug('running %s', _gen_version())
+        format_color='%(log_color)s[%(levelname)s]%(name)s%(reset)s: %(message)s'
+        format_basic='[%(levelname)s] %(name)s: %(message)s'
+        level=logging.DEBUG
     elif args['--verbose']:
-        logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
+        format_color='%(log_color)s%(levelname)s%(reset)s: %(message)s'
+        format_basic='%(levelname)s: %(message)s'  
+        level=logging.INFO
     else:
-        logging.basicConfig(level=logging.WARNING, format='%(levelname)s: %(message)s')
+        format_color='%(log_color)s%(levelname)s%(reset)s: %(message)s'
+        format_basic='%(levelname)s: %(message)s'
+        level=logging.WARNING
         sys.tracebacklimit = 0
+   
+    if use_color_log:
+        formatter = colorlog.ColoredFormatter(format_color)
```

Comment:
I think this needs to use [`TTYColoredFormatter`](https://github.com/borntyping/python-colorlog/blob/1297254a29a4d6a68cc5e0a477a075f4421dd285/colorlog/colorlog.py#L224) instead:

> Blanks all color codes if not running under a TTY.
> This is useful when you want to be able to pipe colorlog output to a file.

This is probably enough, although I don't honestly remember exactly how this stuff works.  But, essentially, we don't want ANSI escape codes if the output is redirected to a file or piped to another program.

Reply:
that did not occur to me before when I implemented this. (I also did not look through the code, I assumed that they would have documented the things in the readme). Ill check this and update it. 

I will also test it when not running under a TTY, Im not quite sure how to test that but Ill see what I can come up with.